build_tools/circle/build_doc.sh

```diff
@@ -143,7 +144,7 @@ if [ -n "$CI_PULL_REQUEST" ]
 then
 	echo "The following documentation files may have been changed by PR #$CI_PULL_REQUEST:"
 	affected=$(affected_doc_paths)
-	echo "$affected" | sed 's|^|* http://scikit-learn.org/circle?'$CIRCLE_BUILD_NUM'/|'
```

Comment:
I don't think that the URL with `scikit-learn.org/circle?` was actually that useful so I went for a straight removal here.

Reply:
I actually find it useful. The plugin seems to work about half the time for me. How do you access the docs?

Reply:
Hm ok I guess the instructions now are good enough. I didn't realize you could just append ``#artifacts``.

Reply:
> The plugin seems to work about half the time for me

Can you describe how it does not work? The button does not show up, clicking on it does not do anything, something different ? I have seen 1. and 2. occasionally which I don't really understand. Refreshing the page (github.com/scikit-learn/scikit-learn/pull/pr_id) tends to "fix" it.

> I didn't realize you could just append #artifacts.

@GaelVaroquaux discovered this, which is quite handy for people who haven't got the necessary rights (or you have not logged in into CircleCI yet)

Reply:
Button doesn't show up. Refreshing sometimes helps. Also, if there's a build running, I'm not sure what happens. Sometimes I want to look at the last completed built.

Reply:
> Button doesn't show up. Refreshing sometimes helps.

Hmm, yeah not sure, note you need to be on the main PR page, e.g. https://github.com/scikit-learn/scikit-learn/pull/10167 and not the diff or the commit view https://github.com/scikit-learn/scikit-learn/pull/10167/files.

> Also, if there's a build running, I'm not sure what happens. Sometimes I want to look at the last completed built.

Right now, the build is always the last one (as found in the CI statuses at the end of the page). If the build is running I think you will get "Artifact not found". This is probably doable with a bit of javascript, but I am not sure this is a good behaviour by default, I feel it's easy to forget that you are not looking at the last commit documentation. You could probably add a tickbox to toggle the behaviour but I will leave that for a rainy day (OK it's raining right now in Paris but it's a bit too late for Javascript anyway).

Reply:
Yeah that's what I thought. Maybe a "see circle ci" greyed out if there there is a thing building and next to it a "last completed built"?

Reply:
(I don't want to bully you into writing more Javascript, though ;)